src/pyhf/pdf.py

```diff
@@ -635,14 +635,21 @@ class Model:
     """The main pyhf model class."""
 
     def __init__(
-        self, spec, modifier_set=None, batch_size=None, validate=True, **config_kwargs
+        self,
+        spec,
+        modifier_set=None,
+        batch_size=None,
+        validate: bool = True,
+        **config_kwargs,
     ):
         """
         Construct a HistFactory Model.
 
         Args:
             spec (:obj:`jsonable`): The HistFactory JSON specification
-            batch_size (:obj:`None` or :obj:`int`): Number of simultaneous (batched) Models to compute.
+            batch_size (:obj:`None` or :obj:`int`): Number of simultaneous (batched)
+            Models to compute.
```

Comment:
```suggestion
             Models to compute.
```

space is just for alignment issues in docs